rust/libnewsboat/src/utils.rs

```diff
@@ -262,6 +264,16 @@ pub fn is_valid_attribute(attribute:  &str) -> bool {
     VALID_ATTRIBUTES.contains(&attribute)
 }
 
+pub fn strwidth( rs_str: &str) -> usize {
+    let control = rs_str.chars().fold( true, |acc, x| acc & !x.is_control());
```

Comment:
@zaowen Did you mean
```rust
let control = rs_str.chars().any(|x| !x.is_control());
```

Reply:
@pickfire Thanks for taking the time to read through the code, but I don't think there is much use in reviewing a long-merged PR. Random improvements are always welcome, so if you spot stuff like this, just change it and submit a new PR.